spec/calendar.html

```diff
@@ -392,6 +405,26 @@ <h1>ISODay ( _dateOrDateTime_ )</h1>
         1. Return 𝔽(_dateOrDateTime_.[[ISODay]]).
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-temporal-isomergefields" aoid="ISOMergeFields">
+      <h1>ISOMergeFields ( _fields_, _additionalFields_ )</h1>
+      <p>
+        The abstract operation ISOMergeFields implements the calendar-specific logic in the `Temporal.Calendar.mergeFields` method for the ISO 8601 calendar.
+      </p>
+      <emu-alg>
+        1. Let _keys_ be ? _additionalFields_.[[OwnPropertyKeys]]().
+        1. If _keys_ contains *"year"*, then
+          1. Perform ? DeletePropertyOrThrow(_fields_, *"era"*).
+          1. Perform ? DeletePropertyOrThrow(_fields_, *"eraYear"*).
+        1. If _keys_ contains *"era"* and *"eraYear"*, then
+          1. Perform ? DeletePropertyOrThrow(_fields_, *"year"*).
+        1. For each element _nextKey_ of _keys_, do
+          1. Let _propValue_ be ? Get(_additionalFields_, _nextKey_).
+          1. If _propValue_ is not *undefined*, then
+            1. Perform ? Set(_fields_, _nextKey_, _propValue_, *true*).
+        1. Return _fields_.
```

Comment:
`Object.Assign` doesn't seem to match this :thinking: and shouldn't ISOMergeFields reject era/eraYear being present at all? (FWIW, validation should already do that, but I haven't specified it yet; however, if that text is removed, Object.Assign would be sufficient assuming validation has already rejected era parameters and ISOMergeFields is called after ToTemporal____Fields has been called on `_fields_` and `_additionalFields_`)

Reply:
> `Object.Assign` doesn't seem to match this

You're right about that, but see a few comments back, this will change.

> shouldn't ISOMergeFields reject era/eraYear being present at all?

Oops, you're right about that also.